pkg/scheduler/factory.go

```diff
@@ -259,7 +258,7 @@ func (c *Configurator) createFromConfig(policy schedulerapi.Policy) (*Scheduler,
 	// CLI configuration.
 	if policy.HardPodAffinitySymmetricWeight != 0 {
 		v := policy.HardPodAffinitySymmetricWeight
-		args.InterPodAffinityArgs = &interpodaffinity.Args{
+		args.InterPodAffinityArgs = &schedulerv1alpha2.InterPodAffinityArgs{
```

Comment:
Use internal version from `"k8s.io/kubernetes/pkg/scheduler/apis/config"`.

Reply:
this is still outstanding, right?

Reply:
Not for this PR. Would it help if I do a follow up PR with the idea I describe in other comments?

Reply:
Please check https://github.com/kubernetes/kubernetes/pull/88585#discussion_r385853451